bin/i18n/i18n_script_utils.rb

```diff
@@ -28,6 +28,24 @@ def self.crowdin_creds
     @crowdin_creds ||= YAML.load_file(CROWDIN_CREDS_PATH).freeze
   end
 
+  # Parses sync options from the command line
+  #
+  # @param options [Hash] the default options to populate
+  # @return [Hash] the parsed options
+  def self.parse_options(argv = ARGV, options: {})
+    options[:testing] = TESTING_BY_DEFAULT if options[:testing].nil?
+
+    OptionParser.new do |parser|
+      parser.on('-t', '--testing', 'Run in testing mode') do
+        options[:testing] = true
+      end
+
+      yield(parser, options) if block_given?
+    end.parse!(argv)
+
+    options
+  end
+
```

Comment:
When I read the description of the PR moving the `option parser` to the `i18n_script_utils` was my first thought. I'm really glad we were thinking the same way!